packages/frames.js/src/getAddressForFid.ts

```diff
@@ -1,45 +1,47 @@
-import { VerificationAddEthAddressMessage } from "@farcaster/core";
 import { createPublicClient, http, parseAbi } from "viem";
 import { optimism } from "viem/chains";
-import { AddressReturnType } from "./types";
-import { bytesToHexString } from "./utils";
+import { AddressReturnType, HubHttpUrlOptions } from "./types";
 
 /**
  * Returns the first verified address for a given `Farcaster` users `fid` if available, falling back to their account custodyAddress
  */
 export async function getAddressForFid<
-  Options extends { fallbackToCustodyAddress?: boolean } | undefined,
+  Options extends
+    | ({ fallbackToCustodyAddress?: boolean } & HubHttpUrlOptions)
+    | undefined,
 >({
   fid,
-  hubClient,
-  options = { fallbackToCustodyAddress: true },
+  options = {},
```

Comment:
replace this line with 
```
{
fallbackToCustodyAddress = true,
hubHttpUrl = "https://nemes.farcaster.xyz:2281"
}
```

and remove lines 21-24 I think thats valid syntax no

Reply:
if the user only specifies one of the properties the other would be undefined, so this won't work as a place to handle multiple fallback values

Reply:
I thought this was avoided that, but all good